source/sync.txt

```diff
@@ -8,21 +12,129 @@
    :titlesonly:
    :hidden:
    
-   What is {+sync+}? </sync/learn>
+   Get Started </sync/get-started>
    Configure and Update Your Data Model </sync/configure-your-data-model>
    Define Data Access Patterns </sync/data-access-patterns>
    Configure Sync </sync/configure-device-sync>
    Handle Errors </sync/error-handling>
    Add Sync to a Local-Only App </sync/local-to-sync>
+   Production Checklist </sync/production-checklist>
    Sync Production Load Testing </sync/production-load-testing>
    Event Library </sync/event-library>
+   Technical Details </sync/details>
+
+.. introduction::
+
+   You're developing a mobile app. Your users want their data saved on the cloud
+   and accessible from their other devices. Network access on a mobile device
+   can be hit or miss, so you write data locally on the device first. A
+   background process then synchronizes the data to the cloud and resolves any
+   conflicting writes.
+ 
+   {+sync+} does all that... and more!
+
+   .. button:: Start Building with a Template Starter App
+      :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+
+.. image:: /images/plp/Spot_AzureBlue_Sync_Tech_RealmSync.png
+   :alt: Atlas Device Sync Illustration
+
+.. kicker:: What It Is
+
+Key Features
+------------
+
+.. card-group::
+   :columns: 3
+   :style: compact
+
+   .. card::
+      :headline: Realm in the Front, MongoDB in the Back
```

Comment:
As much as I think this line is snappy, I'm not sure it works for this case. When I am reading about dev stuff and I hear "front" I think UI, but here we're talking about the client DB. 

Maybe instead this is something like "Realm in the client, MongoDB in the cloud" ?

Reply:
That would rob it of its snappiness. I'm not married to this catchphrase, by the way, but I had to get it out of my system...

"Front" in this case means client (front end), "back" meaning server (back end). I'm not sure I understand (agree with) your specific comment here.

Reply:
Fair. I spend a lot of time talking to my dev friends about dev stuff, and in that circle, "front" generally means "web UI." Perhaps "front end" for a mobile dev is the client and that is a reasonable inference. I was just thinking the cloud/client language is more descriptive/explicit.

Reply:
The reason "front" might refer to "web UI" is because a web UI is a client. Clients are typically considered the front end because it's what the user is actually facing in order to interact with the server (backend).

The phrase "frontend developer" might have a connotation of being someone who specializes in CSS (so, web only), but I think this is just because there's less distinction within the mobile developer space. You would expect a "mobile developer" to be able to UI code.